### PR TITLE
Nucleotide mutations endpoint

### LINKED ISSUE #205: Implement in the `/nucleotideMutations` endpoint
Return only JSON for now, CSV will follow in a separate issue.

### PR DESCRIPTION
**Pull request by fengelniederhammer**
resolves #205

### CODE REVIEW

File: lapis2/src/test/kotlin/org/genspectrum/lapis/controller/LapisControllerTest.kt [hunk 1]
```diff
@@ -40,4 +41,92 @@ class LapisControllerTest(@Autowired val mockMvc: MockMvc) {
             .andExpect(status().isOk)
             .andExpect(jsonPath("\$.count").value(0))
     }
+
+    @Test
+    fun `GET nucleotideMutationProportions without explicit minProportion defaults to 0,05`() {
```
Comment (chaoran-chen) [inline lapis2/src/test/kotlin/org/genspectrum/lapis/controller/LapisControllerTest.kt:46]:
```suggestion
    fun `GET nucleotideMutationProportions without explicit minProportion defaults to 0.05`() {
```

**Review by chaoran-chen — CHANGES_REQUESTED** [review]
Just a few minor comments, looks good otherwise!

File: lapis2/src/test/kotlin/org/genspectrum/lapis/controller/LapisControllerTest.kt (hunk 1 above)
Comment (fengelniederhammer) [reply lapis2/src/test/kotlin/org/genspectrum/lapis/controller/LapisControllerTest.kt:46]: Unfortunately that is not valid Kotlin syntax... That's why I went for the `,` as a workaround. Do you have a better suggestion?

Comment (chaoran-chen) [reply lapis2/src/test/kotlin/org/genspectrum/lapis/controller/LapisControllerTest.kt:46]: oh. Would 5% be valid?

Comment (chaoran-chen) [reply lapis2/src/test/kotlin/org/genspectrum/lapis/controller/LapisControllerTest.kt:46]: 5e-2? 😅

File: lapis2/src/main/kotlin/org/genspectrum/lapis/controller/LapisController.kt [hunk 2]
```diff
@@ -41,7 +48,49 @@ class LapisController(val aggregatedModel: AggregatedModel) {
         @RequestBody
         sequenceFilters: Map<String, String>,
     ): AggregatedResponse {
-        return aggregatedModel.handleRequest(sequenceFilters)
+        return siloQueryModel.aggregate(sequenceFilters)
+    }
+
+    @GetMapping("/nucleotideMutationProportions")
```
[the remark this answers is not in the document]
Comment (fengelniederhammer) [reply lapis2/src/main/kotlin/org/genspectrum/lapis/controller/LapisController.kt]: I shortened it to `/nucleotideMutations`

File: lapis2/src/test/kotlin/org/genspectrum/lapis/controller/LapisControllerTest.kt (hunk 1 above)
Comment (fengelniederhammer) [reply lapis2/src/test/kotlin/org/genspectrum/lapis/controller/LapisControllerTest.kt]: According to IntelliJ, `%` can cause problems on windows, so I went for `5 percent` :+1:

**Review by chaoran-chen — APPROVED** [review]
Thanks!

**Pull request merged**